source/common/common/utility.cc

```diff
@@ -486,6 +486,33 @@ std::string StringUtil::removeCharacters(const absl::string_view& str,
   return absl::StrJoin(pieces, "");
 }
 
+bool StringUtil::consumeLeadingDigits(absl::string_view* s, uint64_t* val) {
+  const char* p = s->data();
+  const char* limit = p + s->size();
+  uint64_t v = 0;
```

Comment:
I think some or all of these single letter variables would benefit from some more letters. Probably v the most: the others' purposes are obvious just from looking at the line they're declared on, but not v.

Reply:
Any ideas what to call it instead of `v`?

Reply:
Maybe `cur_val`, with `new_v` becoming `new_val`, and `val` becoming `out_val`. I would also replace `p` with `ptr`, and something like `input` for `s`. Overall, while I'm not sure if it's particularly codified, I think the (inherited by Envoy) Google style rarely uses single letter variables outside the standard `for` loop `i`.